docs/source/10min_dask_cudf.ipynb

```diff
@@ -0,0 +1,2565 @@
+   "source": [
+    "Selecting rows from index 2 to index 5 from columns 'a' and 'b'."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   a   b\n",
+      "2  2  17\n",
+      "3  3  16\n",
+      "4  4  15\n",
+      "5  5  14\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.loc[2:5, ['a', 'b']])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   a   b\n",
+      "2  2  17\n",
+      "3  3  16\n",
+      "4  4  15\n",
+      "5  5  14\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ddf.loc[2:5, ['a', 'b']].compute())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Selection by Position"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Selecting via integers and integer slices, like numpy/pandas. Note that this functionality is not available for dask_cudf DataFrames."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "a     0\n",
+      "b    19\n",
+      "c     0\n",
+      "Name: 0, dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.iloc[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   a   b\n",
+      "0  0  19\n",
+      "1  1  18\n",
+      "2  2  17\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df.iloc[0:3, 0:2])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also select elements of a `DataFrame` or `Series` with direct index access."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   a   b  c\n",
+      "3  3  16  3\n",
+      "4  4  15  4\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(df[3:5])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3     \n",
+      "4    4\n",
+      "dtype: int64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(s[3:5])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Boolean Indexing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Selecting rows in a `DataFrame` or `Series` by direct Boolean indexing. Boolean indexing on `Series` objects containing nulls is not currently implemented."
```

Comment:
This should be fixed in the latest 0.8 nightlies, we recently merged handling the nullmask in boolean masking 😄 
```suggestion
    "Selecting rows in a `DataFrame` or `Series` by direct Boolean indexing."
```